src/lib/spsp.js

```diff
@@ -36,19 +43,76 @@ const _querySPSP = function * (receiver) {
     .set('Accept', 'application/json')).body
 }
```

Comment:
This should probably validate the response and let you know if it's a valid SPSP receiver